### DISCUSSION TITLE
How do I delete Onion OS from my Miyoo Mini Plus?

### DISCUSSION THREAD

**Discussion by benjifresa** [OnionUI/Onion#806] [Q&A]
I just got Onion OS on my Miyoo Mini Plus until I realized that my version doesn't work for the Onion OS. Does anyone know how to delete this program because I don't have a backup? (Which was stupid of me, you should always make a backup before doing things like this.)

**Comment by XK9274** [comment]
What's not working? If you have a MMP make sure you're using the latest beta build and not 4.1 (It needs to be 4.2 which is found here: https://github.com/OnionUI/Onion/releases/tag/latest). You may also need to update your firmware.

If you do want to delete onion you can just delete all the files from your SD, make sure you keep your Roms etc. 

The main OS runs from the MMP itself not the SD 

Hope this helps!
